opensrp-chw/src/androidTest/java/org/smartregister/chw/activity/wcaro/FamilyMemberTest.java

```diff
@@ -0,0 +1,197 @@
+package org.smartregister.chw.activity.wcaro;
+
+
+import androidx.test.rule.ActivityTestRule;
+
+import org.junit.After;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.smartregister.chw.R;
+import org.smartregister.chw.activity.LoginActivity;
+import org.smartregister.chw.activity.utils.Configs;
+import org.smartregister.chw.activity.utils.Constants;
+import org.smartregister.chw.activity.utils.Order;
+import org.smartregister.chw.activity.utils.OrderedRunner;
+import org.smartregister.chw.activity.utils.Utils;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.clearText;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static androidx.test.espresso.action.ViewActions.typeText;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withHint;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withSubstring;
+
+@RunWith(OrderedRunner.class)
+public class FamilyMemberTest {
+
+    @Rule
+    public ActivityTestRule<LoginActivity> mActivityTestRule = new ActivityTestRule<>(LoginActivity.class);
+
+    private Utils utils = new Utils();
+
+
+    public void setUp() throws InterruptedException{
+        utils.logIn(Constants.WcaroConfigUtils.wCaro_username, Constants.WcaroConfigUtils.wCaro_password);
+        Thread.sleep(5000);
+    }
+
+    @Test
+    @Order(order = 8)
+    public void removeFamilyWithoutReason() throws InterruptedException{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426621)